sdcm/nemesis.py

```diff
@@ -2784,12 +2783,56 @@ def disrupt_network_start_stop_interface(self):  # pylint: disable=invalid-name
             self.target_node.start_network_interface()
             self._wait_all_nodes_un()
 
-    def break_streaming_task_and_rebuild(self, task='decommission'):  # pylint: disable=too-many-statements
+    def _call_disrupt_func_after_expression_logged(self,
+                                                   expression: str,
+                                                   disrupt_func: Callable,
+                                                   disrupt_func_kwargs: dict = None,
+                                                   sleep: int = 1,
+                                                   delay: int = 10,
```

Comment:
in any test we don't change this delay - so it's always 10.
I remember, this delay when was <60 was causing some issues.
See my comment there: https://github.com/scylladb/scylla/issues/10251#issuecomment-1106231160

Reply:
It wasn't causing any problems when it was > 60, probably because operations like repair / rebuild / decommission can complete much faster than that in some cases (e.g. low data scenario at the beginning of test run). In such cases we didn't interrupt anything and just triggered an additional rebuild on the target node. IMHO we actually wan't to interrupt these operations rather sooner than later.

Reply:
that makes sense. Let's try it in battle and see if the mentioned issue is reproducing staidly